src/settings/Settings.ts

```diff
@@ -36,7 +36,7 @@ import { isMac } from '../Keyboard';
 import UIFeatureController from "./controllers/UIFeatureController";
 import { UIFeature } from "./UIFeature";
 import { OrderedMultiController } from "./controllers/OrderedMultiController";
-import { Layout } from "./Layout";
+import {Layout} from "./Layout";
```

Comment:
Please revert the whitespace change, to minimise changes to matrix master

Reply:
Done